department-of-veterans-affairs/caseflow
# Justin/appeals 13220

Resolves [APPEALS-13220](https://vajira.max.gov/browse/APPEALS-13220)

### Description
Add new options to “How are you proceeding?” menu

### Acceptance Criteria
When either adding or editing a CAVC Remand the following selectable options have been added to the "How are you proceeding?" section on the Add CAVC Remand form
Affirmed
Dismissal, other than death
Settlement
When any of the options in AC1 has been selected "Are judgement and mandate dates provided?" appears
Available options y/n
This will default to "yes"
 When any of the options in AC1 have been selected "Same as Court's decision date" will default to unchecked
This is under the label Judgement and mandate, date
When any of the options in AC1 have been selected additional behavior is not triggered when creating the CAVC Remand
These options are for triggering the appearance of the buttons that allow access to the CAVC Dashboard
A new appeal stream should NOT be created
If the feature toggle from [APPEALS-13214](https://vajira.max.gov/browse/APPEALS-13214) is enabled these changes are disabled from being shown


### Testing Plan
1. Log in as a user from CAVC_LIT_SUPPORT
2. Go to "Your Queue"
3. Search for an appeal, such as this veteran ID: 000100002
4. Open the Appeal details page
5. Click on "+Add A CAVC Remans"
6. Click on "Dismissal, other than Death", "Affirmed", and "Settlement".
7. Verify each option shows "Are judgement and mandate dates provided?" with "Yes" preselected.
8. Verify each option shows "Judgement and mandate date" with the "Same as Court's decision date" unchecked by default.


### User Facing Changes

 BEFORE|AFTER
![image](https://user-images.githubusercontent.com/95875751/214887916-0040ab7e-d5a3-4e5c-b197-80f3e6692cc2.png)

 ---|---
![image](https://user-images.githubusercontent.com/95875751/214888385-1e5415d7-5e47-4b3c-bb3d-f552773c9c90.png)
